README.md

```diff
@@ -1 +1,90 @@
-# functional-harness
+## testcafe-functional-harness
+
+Runs functional tests in local browsers and on SauceLabs.
+
+#### What it does
+
+1. Starts the server with the test pages.
```

Comment:
Oops, you should have corrected me - we do not start a server with test pages - we start a TestCafe server, son't we?
```
1. Starts a TestCafe server.
```